Frontend/CO.CDP.OrganisationApp.Tests/Pages/Forms/ChoiceProviderStrategies/ExclusionAppliesToChoiceProviderStrategyTests.cs

```diff
@@ -0,0 +1,63 @@
+    public ExclusionAppliesToChoiceProviderStrategyTests()
+    {
+        mockOrganisationClient = new Mock<IOrganisationClient>();
+        mockUserInfoService = new Mock<IUserInfoService>();
+
+        var orgId = new Guid("0984665e-4a4b-4e9a-88d8-cd97a6fae13e");
+
+        mockUserInfoService.Setup(u => u.GetOrganisationId()).Returns(orgId);
+
+        mockOrganisationClient
+            .Setup(oc => oc.GetOrganisationAsync(It.IsAny<Guid>()))
+            .ReturnsAsync(new CO.CDP.Organisation.WebApiClient.Organisation(null, null, null, null, orgId, null, "Org name", []));
+
+        choiceProviderStrategy = new ExclusionAppliesToChoiceProviderStrategy(mockUserInfoService.Object, mockOrganisationClient.Object);
+    }
+
+    [Fact]
+    public async Task RenderMethodShouldRenderCorrectly_WhenGivenAConnectedOrganisation()
+    {
+        mockOrganisationClient
+            .Setup(oc => oc.GetConnectedEntityAsync(It.IsAny<Guid>(), It.IsAny<Guid>()))
+            .ReturnsAsync(new ConnectedEntity([], "asd", null, ConnectedEntityType.Organisation, false, new Guid(), null, new ConnectedOrganisation(ConnectedOrganisationCategory.AnyOtherOrganisationWithSignificantInfluenceOrControl, [], 3, null, null, "Org name", new Guid(), null), null, null, "Register name", null));
+
+        var result = await choiceProviderStrategy.RenderOption(new Models.FormAnswer() { JsonValue = "{\"id\": \"1083f914-a4c0-4e0b-8650-0b5e46696666\", \"type\": \"connected-entity\"}" });
+
+        result.Should().Be("Org name");
+    }
+
+    [Fact]
+    public async Task RenderMethodShouldRenderCorrectly_WhenGivenAConnectedTrustee()
+    {
+        mockOrganisationClient
+            .Setup(oc => oc.GetConnectedEntityAsync(It.IsAny<Guid>(), It.IsAny<Guid>()))
+            .ReturnsAsync(new ConnectedEntity([], "asd", null, ConnectedEntityType.TrustOrTrustee, false, new Guid(), new ConnectedIndividualTrust(ConnectedIndividualAndTrustCategory.AnyOtherIndividualWithSignificantInfluenceOrControlForTrust, ConnectedPersonType.TrustOrTrustee, [], null, "First name", 4, "Last name", null, new Guid(), null), null, null, null, "Register name", null));
+
+        var result = await choiceProviderStrategy.RenderOption(new Models.FormAnswer() { JsonValue = "{\"id\": \"1083f914-a4c0-4e0b-8650-0b5e46696666\", \"type\": \"connected-entity\"}" });
+
+        result.Should().Be("First name Last name");
+    }
+
+    [Fact]
+    public async Task RenderMethodShouldRenderCorrectly_WhenGivenAConnectedIndividual()
+    {
+        mockOrganisationClient
+            .Setup(oc => oc.GetConnectedEntityAsync(It.IsAny<Guid>(), It.IsAny<Guid>()))
+            .ReturnsAsync(new ConnectedEntity([], "asd", null, ConnectedEntityType.Individual, false, new Guid(), new ConnectedIndividualTrust(ConnectedIndividualAndTrustCategory.PersonWithSignificantControlForIndividual, ConnectedPersonType.Individual, [], null, "First name", 4, "Last name", null, new Guid(), null), null, null, null, "Register name", null));
```

Comment:
There's lots of various kinds of ConnectedEntity objects created in between test cases. How do they differ?

Can we add a factory method to create ConnectedEntity, parametrise it, and pass only details that are relevant for each test case while hiding unrelevant details? This would make these test a bit more readable.

I can provide an example if needed.

Reply:
yeah that makes sense, I can do that

Reply:
Done